packages/react-dom/src/client/ToStringValue.js

```diff
@@ -35,3 +37,42 @@ export function getToStringValue(value: mixed): ToStringValue {
       return '';
   }
 }
+
+/**
+ * Returns true only if Trusted Types are available in global object and the value is a trusted type.
+ */
+let isTrustedTypesValue: (value: any) => boolean;
+// $FlowExpectedError - TrustedTypes are defined only in some browsers or with polyfill
+if (typeof trustedTypes !== 'undefined') {
+  isTrustedTypesValue = (value: any) =>
+    trustedTypes.isHTML(value) ||
+    trustedTypes.isScript(value) ||
+    trustedTypes.isScriptURL(value) ||
+    // TrustedURLs are deprecated and will be removed soon: https://github.com/WICG/trusted-types/pull/204
```

Comment:
So we even want to support them then?

Reply:
I think at least for now we should. Both chrome and the polyfill currently support them and if we don't, trying out the integration will not work for the users (because the values will be stringified). They would need to create a default policy that would bless all URLs.